common-fragment-V1/src/test/java/org/pentaho/hadoop/shim/common/DistributedCacheUtilImplTest.java

```diff
@@ -311,6 +291,32 @@ public void stageForCache_destination_no_overwrite() throws Exception {
     }
   }
 
+  @Test
+  public void stageForCache_exclude_file() throws Exception {
+    DistributedCacheUtilImpl ch = new DistributedCacheUtilImpl();
+
+    Configuration conf = new Configuration();
+    FileSystem fs = DistributedCacheTestUtil.getLocalFileSystem( conf );
+
+    FileObject source = DistributedCacheTestUtil.createTestFolderWithContent();
+    try {
+      Path root = new Path( "bin/test/stageForCache_exclude_file" );
+      Path dest = new Path( root, "dest" );
+
+      try {
+        ch.stageForCache( source, fs, dest, "foo,jar2,bar", false, true );
```

Comment:
Can you write/modify a test that verifies that the argument "excludePluginFileNames" does infact remove multiple files if listed?

so like 
`ch.stageForCache( source, fs, dest, "foo,jar2,bar,jar3", false, true );`

just to verify that it removes more than one jar file and it doesn't remove all jar files.

Reply:
Can do